aytests/services.sh

```diff
@@ -5,26 +5,22 @@
 # Services have been enabled/disabled in the first installation stage
 # because on CaaSP there is no second stage anymore.
 
-OK=1
+zgrep "systemctl enable sshd.service" /var/log/YaST2/y2log-1.gz
```

Comment:
np: What about putting `/var/log/YaST2/y2log-1.gz`? Sorry, I should say you it before.